.gitignore

```diff
@@ -7,3 +7,7 @@ vocabs/v1/test.ttl
 .DS_Store
 .idea
 unresolveable-test-dids.json
+
+# This file is ignored because it is
+# generated by the CI system.
+methods/index.json
```

Comment:
@msporny adding this file here so to does not pollute PR reviews, however, you can run the commands the CI will run locally, and see that is is generated properly...

Reply:
I'm fine w/ this.